boringssl-static/pom.xml

```diff
@@ -196,15 +196,17 @@
                             <then>
                               <!-- On *nix, add ASM flags to disable executable stack -->
                               <property name="cmakeAsmFlags" value="-Wa,--noexecstack" />
-                              <property name="cmakeCFlags" value="-std=c99 -O3 -fno-omit-frame-pointer" />
+                              <!-- Use -DOPENSSL_C11_ATOMIC so we replace most of the locking code with atomics-->
+                              <property name="cmakeCFlags" value="-O3 -fno-omit-frame-pointer -DOPENSSL_C11_ATOMIC" />
                               <!-- We need to define __STDC_CONSTANT_MACROS and __STDC_FORMAT_MACROS when building boringssl on centos 6 -->
                               <property name="cmakeCxxFlags" value="-O3 -fno-omit-frame-pointer -Wno-error=maybe-uninitialized -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS" />
                             </then>
                           </elseif>
                           <else>
                             <!-- On *nix, add ASM flags to disable executable stack -->
                             <property name="cmakeAsmFlags" value="-Wa,--noexecstack" />
```

Comment:
Not for this PR, and there's no harm in leaving it along but, FYI, `--noexecstack` should (hopefully?) not be needed here. Our CMake files add it and our assembly files include the `.note.GNU-stack` section on ELF targets. (As far as I'm aware, non-ELF targets don't need it?)

My understanding of this mess comes from https://www.airs.com/blog/archives/518

Reply:
I will have a look as a followup ... thanks